examples/post_training_quantization/tensorflow/mobilenet_v2/main.py

```diff
@@ -0,0 +1,189 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import os
+import re
+import subprocess
+from pathlib import Path
+from typing import List, Optional
+
+import openvino.runtime as ov
+import tensorflow as tf
+import tensorflow_datasets as tfds
+from keras.utils import data_utils
+from openvino.tools import mo
+from tqdm import tqdm
+
+import nncf
+
+ROOT = Path(__file__).parent.resolve()
+WEIGHTS_URL = 'https://huggingface.co/alexsu52/mobilenet_v2_imagenette/resolve/main/tf_model.h5'
+DATASET_CLASSES = 10
+
+
+def validate(model: ov.Model, 
+             val_loader: tf.data.Dataset) -> tf.Tensor:
+    compiled_model = ov.compile_model(model)
+    output = compiled_model.outputs[0]
+
+    metric = tf.keras.metrics.CategoricalAccuracy(name='acc@1')
+    for images, labels in tqdm(val_loader):
+        pred = compiled_model(images)[output]
+        metric.update_state(labels, pred) 
+   
+    return metric.result()
+
+
+def run_benchmark(model_path: str, shape: Optional[List[int]] = None, 
+                  verbose: bool = True) -> float:
+    command = f'benchmark_app -m {model_path} -d CPU -api async -t 15'
+    if shape is not None:
+        command += f' -shape [{",".join(str(x) for x in shape)}]'
+    cmd_output = subprocess.check_output(command, shell=True)
+    if verbose:
+        print(*str(cmd_output).split('\\n')[-9:-1], sep='\n')
+    match = re.search(r"Throughput\: (.+?) FPS", str(cmd_output))
+    return float(match.group(1))
+
+
+def get_model_size(ir_path: str, m_type: str = 'Mb', 
+                   verbose: bool = True) -> float:
+    xml_size = os.path.getsize(ir_path)
+    bin_size = os.path.getsize(ir_path.replace('xml', 'bin'))
```

Comment:
Just to be confident
```suggestion
    bin_size = os.path.getsize(os.path.splitext(ir_path)[0] + 'bin')
```

Reply:
done